CHANGELOG.md

```diff
@@ -21,6 +22,15 @@ I open my heart to share this component w/ you guys, buy I don't have much free
 
 I will upgrade this pack to any angular major version as soon as possible. Unfortunately I can't replicate new features to old compatibility versions. But you can fork this repo and port does features.
 
+## 13.0.1
+* Bump Karma
+* Bump Jasmine
+* Bump RXJS to 7.4.0
+* Bump Types dependencies
+* Clean up deprecated testing variables
+* Fix typos and duplicated variables
+* Run TSLint and clean up code
```

Comment:
Most of those look like `chores` and there is no need to include them in changelog.